src/content/docs/errors-inbox/errors-email-notifications.mdx

```diff
@@ -41,6 +41,6 @@ Click the **Unsubscribe** link at the bottom of the email notification.
 
 ### Resubscribe to email notifications [#resubscribe]
 
-If you've unsubscribed to email notifications and decide you need to resubscribe, find one your old errors inbox email notifications and click the *Unsubscribe preferences** link.
+If you've unsubscribed to email notifications and decide you need to resubscribe, find one your old errors inbox email notifications and click the **Unsubscribe preferences** link.
 
-The email notification titles start with "[Errors Inbox]".
+The email notification titles start with "**[Errors Inbox]**".
```

Comment:
@brnhensley I feel like either all of it has to be bolded (if the quotes ("") appear in the UI), or none of (if it's not a clickable UI element).

Reply:
@ally-sassman that's the text in the email, I'd assume there are no `"`s in there. I have no strong opinions about how it should look. Whatever you think is best is fine